requirements.txt

```diff
@@ -1,5 +1,7 @@
-py-cord==2.3.2
+asgiref==3.6.0
+openai==0.25.0
 Pillow==9.3.0
+py-cord==2.3.2
```

Comment:
In the future to update dependencies, I would update here, and in the pyproject, starting at lines 20, is that correct?